packages/cozy-realtime/src/SubscriptionList.js

```diff
@@ -59,10 +59,14 @@ export default class SubscriptionList {
     if (requireDoubleUnsubscriptions) {
       // remove only the first matching
       const found = this.subscriptions.findIndex(s => isEqual(s, subscription))
-      if (found >= 0) pullAt(this.subscriptions, found)
+      if (found >= 0) {
+        pullAt(this.subscriptions, found)
+        return
+      }
     } else {
       // remove all matching
-      remove(this.subscriptions, s => isEqual(s, subscription))
+      const removed = remove(this.subscriptions, s => isEqual(s, subscription))
+      if (removed) return
```

Comment:
Haven't run the code, but I  presume `if (removed)` is always true since `remove()` returns an array?